cli/introspection/src/commands/Init.ts

```diff
@@ -6,30 +6,17 @@ import chalk from 'chalk'
 import { printError } from '../prompt/utils/print'
 import { highlightTS, link } from '@prisma/sdk'
 
-export const defaultSchema = `// This is your Prisma schema file,
+export const defaultSchema = `// This is your Prisma schema file
```

Comment:
This is not grammatically not correct. Either have a `.`, `,` or capitalize the next line.